ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -1453,6 +1453,8 @@ CO2M:
   - MONITORING
   opt_uses:
   - high_zone_air_co2_concentration_alarm
+  - high_limit_zone_co2_concentration_setpoint
+  - failed_alarm
```

Comment:
This point would refer to if the general type has failed and should belong on the `SS` for the entity type in most cases. If you are wanting a point to indicate if the co2 sensor has failed then we would need to add that as a new field.

Reply:
removed thank you